src/rest/APIRequest.js

```diff
@@ -1,10 +1,7 @@
 const querystring = require('querystring');
-const snekfetch = require('snekfetch');
-const https = require('https');
+const superagent = require('superagent');
 const { browser, UserAgent } = require('../util/Constants');
 
-if (https.Agent) var agent = new https.Agent({ keepAlive: true });
```

Comment:
this is pretty sad :(

Reply:
I couldn't find any way to do it via superagent, so I assume no support.